### PR TITLE
[FR] Update `osu!_Community_Meetings`

### PR DESCRIPTION
**Pull request by DoryanR**
Proposal to update the French translation of the wiki page `osu!_Community_Meetings`.

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)

*Feel free to merge if the delay is too long for a review by a native.*

### CODE REVIEW

File: wiki/osu!_Community_Meetings/fr.md [hunk 1]
```diff
@@ -6,11 +6,11 @@ Il y a longtemps, dans le [passé lointain du jeu](https://twitter.com/ppy/statu
...
 
-Le prochain osu! community meeting est prévu pour le : **Le 28 novembre 2021 (dimanche 01:00 UTC)**. Les réunions ont généralement lieu toutes les deux semaines, les rappels étant annoncés sur le [compte Twitter de peppy](https://twitter.com/ppy) environ un jour avant.
+Le prochain osu! community meeting est prévu pour le : **11 décembre 2021 (Samedi 21:00 UTC)**. Les réunions ont généralement lieu toutes les deux semaines, les rappels étant annoncés sur le [compte Twitter de peppy](https://twitter.com/ppy) environ un jour avant.
 
 ## S'impliquer
 
-Les osu! community meetings sont actuellement organisées sur une scène Discord sur le Discord [osu!dev](https://discord.gg/ppy), et diffusés en simultané sur [la chaîne Twitch de peppy](https://www.twitch.tv/ppy). Chaque meeting dure environ une heure et a lieu un dimanche sur deux à 01:00 UTC. 
+Les osu! community meetings sont actuellement organisées sur une scène Discord sur le Discord [osu!dev](https://discord.gg/ppy), et diffusés en simultané sur [la chaîne Twitch de peppy](https://www.twitch.tv/ppy). Chaque meeting est actuellement prévue pour durer environ une heure et a lieu un week-end sur deux.
```
Comment (Ganondorf-1) [inline wiki/osu!_Community_Meetings/fr.md:13]:
```suggestion
Les osu! community meetings sont actuellement organisés sur une scène Discord sur le Discord [osu!dev](https://discord.gg/ppy), et diffusés en simultané sur [la chaîne Twitch de peppy](https://www.twitch.tv/ppy). Chaque meeting est actuellement prévu pour durer environ une heure et a lieu un week-end sur deux.
```

File: wiki/osu!_Community_Meetings/fr.md [hunk 2]
```diff
@@ -6,11 +6,11 @@ Il y a longtemps, dans le [passé lointain du jeu](https://twitter.com/ppy/statu
 
 Actuellement, les réunions de la communauté osu! sont présidées par [peppy](https://osu.ppy.sh/users/2), [Ephemeral](https://osu.ppy.sh/users/102335) et [smoogipoo](https://osu.ppy.sh/users/1040328).
 
-Le prochain osu! community meeting est prévu pour le : **Le 28 novembre 2021 (dimanche 01:00 UTC)**. Les réunions ont généralement lieu toutes les deux semaines, les rappels étant annoncés sur le [compte Twitter de peppy](https://twitter.com/ppy) environ un jour avant.
+Le prochain osu! community meeting est prévu pour le : **11 décembre 2021 (Samedi 21:00 UTC)**. Les réunions ont généralement lieu toutes les deux semaines, les rappels étant annoncés sur le [compte Twitter de peppy](https://twitter.com/ppy) environ un jour avant.
```
Comment (Ganondorf-1) [inline wiki/osu!_Community_Meetings/fr.md:9]:
```suggestion
Le prochain osu! community meeting est prévu pour le : **11 décembre 2021 (Samedi 21:00 UTC)**. Les meetings ont généralement lieu toutes les deux semaines, les rappels étant annoncés sur le [compte Twitter de peppy](https://twitter.com/ppy) environ un jour avant.
```

**Review by Ganondorf-1 — CHANGES_REQUESTED** [review]

File: wiki/osu!_Community_Meetings/fr.md [hunk 3]
```diff
@@ -6,11 +6,11 @@ Il y a longtemps, dans le [passé lointain du jeu](https://twitter.com/ppy/statu
 
 Actuellement, les réunions de la communauté osu! sont présidées par [peppy](https://osu.ppy.sh/users/2), [Ephemeral](https://osu.ppy.sh/users/102335) et [smoogipoo](https://osu.ppy.sh/users/1040328).
```
Comment (Ganondorf-1) [inline wiki/osu!_Community_Meetings/fr.md:7]:
```suggestion
Actuellement, les osu! community meetings sont présidés par [peppy](https://osu.ppy.sh/users/2), [Ephemeral](https://osu.ppy.sh/users/102335) et [smoogipoo](https://osu.ppy.sh/users/1040328).
```

**Review by Ganondorf-1 — CHANGES_REQUESTED** [review]
Forgot this; should be good to go once this change is made

**Review by TicClick — APPROVED** [review]

**Pull request merged**